src/dealer-list.js

```diff
@@ -44,18 +44,27 @@ const DealerCard = (dealer, key, onDealerClicked, theme) => (
     </Dealer>
 );
 
+const filterByOnlineDealer = dealer => {
+  if (dealer.online) {
```

Comment:
this is fine but could be more concise if you just do `return dealer.online` - when you're passing that second argument to `_.filter` you don't have to return the whole object, just a truthy value indicating whether to include this item in the result set.